logstreams/src/main/java/io/zeebe/logstreams/processor/ReProcessingStateMachine.java

```diff
@@ -106,8 +99,10 @@ public static ProcessingStateMachineBuilder builder() {
 
   private final ZeebeDb zeebeDb;
   private final RetryStrategy updateStateRetryStrategy;
+  private final RetryStrategy processRetryStrategy;
 
   private final BooleanSupplier abortCondition;
+  private final Set<Long> blacklist = new HashSet<>();
```

Comment:
I would like to rename the field to `failedEventPositions` which make it more clear how it is used.